### DISCUSSION TITLE
overlay scroll disable opencpn

### DISCUSSION THREAD

**Discussion by shlandrews** [bareboat-necessities/lysmarine_gen#418] [Q&A]
I have the following in gtk3 settings.ini

...
gtk-overlay-scrolling=false
gtk-primary-button-warps-slider=false

however in opencpn I still get overlay scrolling happening, which makes it difficult to access some menus etc, such as this on where trying to update SK key menu in screenshot.

perhaps this is a question for the opencpn forums and not bbn related.

![Screenshot_2024-05-13_17-44-30](https://github.com/bareboat-necessities/lysmarine_gen/assets/63079119/6168dd4d-7431-46e5-8fda-7d56d5c72b0e)

**Comment by mgrouch** [comment]
How does it prevent you from accessing anything?
You should be able to touch/click what you need and scrollbar would disappear

**Reply by shlandrews** [reply]
It's what is behind the scroll bar, for instance the ... menu access on the SK Key, when you try hover it to click, the scroll appears and only then have access to the scroll. Of course others which are just a big drop down menu bar then no problem.

**Reply by mgrouch** [reply]
I’d say it’s poor UI design of that plugin in OpenCPN. It should allow margin for a scrollbar. You can report it in issues for the SignalK plugin GitHub project.

**Reply by shlandrews** [reply]
Thanks for this, I will raise the question there. I don't get it on larger screens so it is something with this 10" screen forcing the scroll to hover in opencpn rather than fixed. Weird that gtk-overlay-scrolling=false already in the gtk3 settings does not override it in opencpn.
